### PR TITLE
Vulcan: Style images

### PR DESCRIPTION
**Pull request by andyg0808**
This adds some styles for images which should make them align according to our image configurations.

It also adds styling for videos, which should be visible once https://github.com/iFixit/react-commerce/pull/1427 is merged.

## QA Notes
Please check that images seem to look reasonable under their various styling options.

You might find http://localhost:3000/Vulcan/troubleshooting-test helpful for that.

Closes https://github.com/iFixit/ifixit/issues/46769

### CODE REVIEW

**Comment by deltuh-vee** [comment]
Is the right margin for the boxes around these images supposed to look like this?
image

File: frontend/templates/troubleshooting/prerendered.tsx [hunk 1]
```diff
@@ -72,6 +72,41 @@ const renderStyles: SystemStyleObject = {
...
+         clear: 'right',
+      },
+   },
+
+   '.imageBox_center': {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      '>img': {
+         //margin: '0 auto',
```
Comment (ardelato) [inline frontend/templates/troubleshooting/prerendered.tsx:106]: Should this be commented out?

**Review by ardelato — COMMENTED** [review]
Are we trying to completely match the image configurations we currently have on iFixit? If so, then we need to add bottom and top margins for each `imageBox > img` as well as fit the images on their own respective line.

image

File: frontend/templates/troubleshooting/prerendered.tsx (hunk 1 above)
Comment (andyg0808) [reply frontend/templates/troubleshooting/prerendered.tsx:106]: It should really be deleted; I'm using flexbox instead of margins for this one.

**Comment by andyg0808** [comment]
@ardelato Good point; I think it should be closer now.

**Review by jyee27 — DISMISSED** [review]
CR ☕ through [b4302e2](https://github.com/iFixit/react-commerce/pull/1438/commits/b4302e28e64e273e2b4e8b54a31df2afcb2f6d1e)

**Comment by erinemay** [comment]
dev_block ❔ @andyg0808, I'm seeing a lot of these images with inline text now.

**Comment by andyg0808** [comment]
un_dev_block :+1: 
It's tricky: the images are actually supposed to do that as far as I know; it's just that there are YouTube videos below, and in the original styling, those clear floats, which keeps the text from coming up here. I've configured the new `lite-youtube` elements to clear floats now, so it'll look as-expected, but I don't think it's actually relevant one way or the other.

**Comment by erinemay** [comment]
Oh interesting. I see that. I made a copy troubleshooting page and pulled out the youtube videos below.
https://www.cominor.com/Wiki/erins_troubleshooting_test
image

**Review by jyee27 — APPROVED** [review]
CR ☕ through [`5faaac7` (#1438)](https://github.com/iFixit/react-commerce/pull/1438/commits/5faaac7dea9d5a458fa9fa958af7cb907c9f9ff3)

**Review by jyee27 — APPROVED** [review]
CR ☕ through [`fc84d15` (#1438)](https://github.com/iFixit/react-commerce/pull/1438/commits/fc84d152130adb2b3e4df853cef9fd969287b8b2)

**Comment by k0rvusk0r4x** [comment]
Still looking good.

QA 😺

**Pull request merged**